plugins/zenoh-plugin-storage-manager/src/replica/snapshotter.rs

```diff
@@ -235,12 +242,15 @@ impl Snapshotter {
             drop(log);
         } else {
             let mut log = replica_data.stable_log.write().await;
-            let deleted = (*log).insert(key, ts);
+            let deleted = (*log).insert(key.clone(), ts);
             if deleted.is_some() {
```

Comment:
Same comment than above: better to replace `if x.is_some() { ... x.unwrap() ... }`
with `if let Some(y) = x { ... }`